random_word/random_word.py

```diff
@@ -134,8 +134,43 @@ def word_of_the_day(self, **kwargs):
             definitions = result['definitions']
             return json.dumps({
                 'word': word,
-                'definations': definitions
+                'definitions': definitions
             })
         else:
             raise Exception(
                 "Error occured, No result found. If you think this was a mistake than raise issue at {}".format(self.issue_url))
+
+    def get_word_definitions(self, **kwargs):
+        """Returns a single random word
+
+        Args:
+            word, str: Word to return definitions for (required)
+            limit, int: Maximum number of results to return (optional)
+            partOfSpeech, str: CSV list of part-of-speech types (optional)
+            includeRelated, str: Return related words with definitions (optional)
+            sourceDictionaries, list: Source dictionary to return definitions from. (optional)
+            useCanonical, str: If true (default: false) will try to return the correct word root (optional)
+            includeTags, str: Return a closed set of XML tags in response (optional)
+
+        Returns: String, Definitions
+        """
+
+        url = "https://api.wordnik.com/v4/words.json/randomWord?"
```

Comment:
Have you read the documentation? as this is not the calling endpoint.
Calling endpoint is `https://api.wordnik.com/v4/word.json/mistake/definitions?&api_key=YOURAPIKEY` without any parameters